openedx/core/djangoapps/schedules/templates/schedules/edx_ace/upgradereminder/email/body.html

```diff
@@ -0,0 +1,85 @@
+{% extends 'schedules/edx_ace/common/base_body.html' %}
+{% load i18n %}
+
+{% block preview_text %}
+    {% if course_ids|length > 1 %}
+        {% blocktrans trimmed %}
+            We hope you are enjoying learning with us so far in {{ course_name }}, and other courses on
+            {{ platform_name }}! A verified certificate will allow you to highlight your new knowledge and
+            skills. It's official, and easily shareable.
+
+            Upgrade by {{ user_schedule_upgrade_deadline_time }}.
+        {% endblocktrans %}
+    {% else %}
+        {% blocktrans trimmed %}
+            We hope you are enjoying learning with us so far in {{ course_name }}! A verified certificate
+            will allow you to highlight your new knowledge and skills. It's official, and easily shareable.
+
+            Upgrade by {{ user_schedule_upgrade_deadline_time }}.
+        {% endblocktrans %}
+    {% endif %}
+{% endblock %}
+
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>{% trans "Upgrade now" %}</h1>
+
+            {% if course_ids|length > 1 %}
+                <p>
+                    {% blocktrans trimmed %}
+                        We hope you are enjoying learning with us so far in <strong>{{ course_name }}</strong>, and
+                        other courses on {{ platform_name }}! A verified certificate will allow you to highlight your
+                        new knowledge and skills. It's official, and easily shareable.
+                    {% endblocktrans %}
+                </p>
+                <p>
+                    {% blocktrans trimmed %}
+                        Upgrade by <strong>{{ user_schedule_upgrade_deadline_time }}</strong>.
+                    {% endblocktrans %}
+                </p>
+            {% else %}
```

Comment:
Maybe just remove this for now, since we aren't doing grouping?

Reply:
done!